### DISCUSSION TITLE
How to start?

### DISCUSSION THREAD

**Discussion by burasuk** [serversideup/docker-php#309] [Q&A]
After `docker compose up` i have error:

`PHP Fatal error:  Uncaught Error: Failed opening required '/var/www/html/vendor/autoload.php'  `

and container is down, because composer install don't run.

What am i doing wrong?

`Dockerfile`

```
FROM serversideup/php:8.2-fpm-nginx

ARG USER_ID
ARG GROUP_ID
ENV SSL_MODE=off

COPY --chown=www-data:www-data . /var/www/html
RUN composer install --no-dev --ignore-platform-reqs --optimize-autoloader --prefer-dist --no-ansi --no-interaction --no-progress --no-scripts
```

**Comment by DvDty** [comment]
How does your docker-compose file look?

**Reply by jaydrogers** [reply]
Adding to @DvDty's comment (thanks for chiming in).

1. Be sure to use the `beta` images if you're getting started. That will give you the latest features
2. Here is a good example running a Laravel application https://github.com/serversideup/financial-freedom
